.github/workflows/ci-release.yaml

```diff
@@ -14,12 +14,6 @@ on:  # yamllint disable-line rule:truthy
       - Pipfile
       - Pipfile.lock
 
-# If multiple pull requests are merged before a release is done, keep building the current one and then build the next
```

Comment:
This is no longer relevant as we now require merge queues (plus it didn't work anyway).